# @author: Matteo Caruso
# @email: matteo.caruso@phd.units.it
# @email: matteo.caruso1993@gmail.com


dinamixel_topic_name: "cmd_vel_motors"
ctrl_rate: 10 #Hz


pid_values_velocity_p: 50
pid_values_velocity_d: 0.005
pid_values_velocity_i: 5


pid_values_position_p: 50
pid_values_position_d: 5
pid_values_position_i: 5


publish_real_tf: true